README.md

```
@@ -137,7 +137,7 @@ Static code checks and tests can be executed by running `make verify`. We are us
 
 Feedback and contributions are always welcome!
 
-Please report bugs or suggestions as [GitHub issues](https://github.com/gardener/gardener-extension-runtime-gvisor/issues) or reach out on [Slack](https://gardener-cloud.slack.com/) (join the workspace [here](https://gardener.cloud/community/community-bio/)).
+Please report bugs or suggestions as [GitHub issues](https://github.com/gardener/gardener-extension-runtime-gvisor/issues) or reach out on [Slack](https://gardener-cloud.slack.com/) (join the workspace [here](https://gardener.cloud/community/#get-connected)).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)